Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesDelegate.cs

```diff
@@ -18,10 +20,17 @@ public class CrashesDelegate
         static NativeShouldProcessErrorReportDelegate del;
         static Crashes.ShouldProcessErrorReportHandler externalHandler = null;
 
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+        delegate IntPtr NativeGetErrorAttachmentsDelegate(IntPtr report);
+        static NativeGetErrorAttachmentsDelegate delGetAttachments;
+        static Crashes.GetErrorAttachmentstHandler getErrorAttachmentsHandler = null;
```

Comment:
Please add access modifiers and fix names here and above

Reply:
Ok